### PR TITLE
add article that links to statistics courses

### PR DESCRIPTION
**Pull request by hansvancalster**
A long overdue task on my to do list...

### CODE REVIEW

File: content/articles/inbo_stats_courses/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+While we do plan to periodically repeat these courses for new employees and as a refresher, the course material can always be consulted and used for self-teaching or as a reference.
+
+We chose to maintain the links to googledrive folders with the course material on our intranet and therefore limit ourselves here to just point you to that link.
+You can find the overview [here](https://sites.google.com/a/inbo.be/intranet/ondersteuning/onderzoek/bmk/lessenreeks-en-tutorials).
+This course material is only accessible for INBO employees.
+
+Do note, however, that more recently developed in-house statistics courses will most likely end-up being part of this public website.
+An example is the [follow-up INLA workshop](https://inbo.github.io/tutorials/tutorials/r_inla/).
+Moreover, we have several short tutorials about very specific statistical topics.
```
Comment (ElsLommelen) [inline content/articles/inbo_stats_courses/index.md:25]:
```suggestion
Moreover, we have several short tutorials on very specific statistical topics.
```
I'm not sure which is the best, but this sounds better for me.

File: content/articles/inbo_stats_courses/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+tags: ["r", "INLA", "mixed model", "uncertainty", "explorative data analysis", "analysis of variance", "generalized linear regression", "multivariate statistics"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+During the course of the years, we have taught many statistics courses at the Research Institute for Nature and Forest (INBO), both for scientists and technicians.
+Most of these courses have been developed and taught in-house.
+Some (mostly advanced) courses were taught in-house by external teachers.
```
Comment (ElsLommelen) [inline content/articles/inbo_stats_courses/index.md:15]:
```suggestion
Some (mostly advanced) courses were taught in-house by external teachers.
```
I'm not sure if we should mention external courses here as well...

**Review by ElsLommelen — APPROVED** [review]
I only reviewed the commits and changes, and it looks good to me.  I added some minor suggestions to have a look at.

File: content/articles/inbo_stats_courses/index.md (hunk 2 above)
Comment (hansvancalster) [reply content/articles/inbo_stats_courses/index.md:15]: I think that is OK. I will leave it as is.

File: content/articles/inbo_stats_courses/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+While we do plan to periodically repeat these courses for new employees and as a refresher, the course material can always be consulted and used for self-teaching or as a reference.
+
+We chose to maintain the links to googledrive folders with the course material on our intranet and therefore limit ourselves here to just point you to that link.
+You can find the overview [here](https://sites.google.com/a/inbo.be/intranet/ondersteuning/onderzoek/bmk/lessenreeks-en-tutorials).
+This course material is only accessible for INBO employees.
+
+Do note, however, that more recently developed in-house statistics courses will most likely end-up being part of this public website.
```
Comment (florisvdh) [inline content/articles/inbo_stats_courses/index.md:23]:
```suggestion
Do note, however, that more recently developed in-house statistics courses will most likely end up being part of the public tutorials website you are visiting.
```

**Review by florisvdh — APPROVED** [review]
Thanks for adding this @hansvancalster :clap:. Best add yourself as author of the tutorial. One minor comment below.

**Pull request merged**